src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperDescriptionFactory.cs

```diff
@@ -225,16 +226,20 @@ internal static bool TryExtractSummary(string documentation, out string summary)
                 return false;
             }
 
-            var summaryTagStart = documentation.IndexOf(summaryStartTag, StringComparison.OrdinalIgnoreCase);
-            if (summaryTagStart == -1)
-            {
-                summary = null;
-                return false;
-            }
+            documentation = documentation.Trim(new char[] { '\n', '\r' });
 
+            var summaryTagStart = documentation.IndexOf(summaryStartTag, StringComparison.OrdinalIgnoreCase);
             var summaryTagEndStart = documentation.IndexOf(summaryEndTag, StringComparison.OrdinalIgnoreCase);
-            if (summaryTagEndStart == -1)
+            if (summaryTagStart == -1 || summaryTagEndStart == -1)
             {
+                // A really stupid but cheap way to check if this is XML
```

Comment:
> stupid

Policheck violation?

Reply:
Does it count if I'm insulting myself? (Will fix.)